builds/e2e/e2e.yaml

```diff
@@ -212,6 +212,67 @@ jobs:
     timeoutInMinutes: 120
 
     steps:
+    - bash: |
+        set -e
+
+        # Install PowerShell dependencies
+        sudo apt-get update
+        sudo apt-get install -y liblttng-ust0 jq
+
+        # Get the PowerShell tarball
+        release_id=$(curl -sSH 'Accept: application/vnd.github.v3+json' https://api.github.com/repos/powershell/powershell/releases/latest | jq '.id')
+        url=$(curl -sSH 'Accept: application/vnd.github.v3+json' https://api.github.com/repos/powershell/powershell/releases/$release_id/assets | jq -r ".[] | select(.name | contains(\"linux-arm64.tar.gz\")) | .browser_download_url")
+        echo "$release_id"
+        echo "$url"
+        file=$(basename $url)
+        curl -sSL $url -o /tmp/$file
+
+        # Install PowerShell
+        sudo mkdir -p /opt/microsoft/powershell/7
+        sudo tar zxf /tmp/$file -C /opt/microsoft/powershell/7
+        sudo chmod +x /opt/microsoft/powershell/7/pwsh
+        sudo ln -s /opt/microsoft/powershell/7/pwsh /usr/bin/pwsh
+      displayName: Install pwsh
+
+    - bash: |
+        set -e
+
+        # Install .NET Core dependencies
+        sudo apt-get update
+        sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+            libc6 libgcc1 libgssapi-krb5-2 libicu66 libssl1.1 libstdc++6 zlib1g
+
+        # Install .NET Core
+        curl -sSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --channel 3.1
+        sudo ln -s $HOME/.dotnet/dotnet /usr/bin/dotnet
+      displayName: Install .NET Core
+
+    - task: DownloadBuildArtifacts@0
+      displayName: Download Resource Build Images
+      inputs:  
+        buildType: specific
+        project: $(resources.pipeline.images.projectID)
+        pipeline: $(resources.pipeline.images.pipelineName)
+        buildVersionToDownload: specific
+        buildId: $(resources.pipeline.images.runID)
+        downloadType: single
+        artifactName: $(az.pipeline.images.artifacts)
+        allowPartiallySucceededBuilds: true
+        itemPattern: $(az.pipeline.images.artifacts)/artifactInfo.txt
+
+    - bash: |
+        set -e
+
+        # Install Moby
+        curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list > ./microsoft-prod.list
+        sudo cp ./microsoft-prod.list /etc/apt/sources.list.d/
+        curl https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.gpg
+        sudo cp ./microsoft.gpg /etc/apt/trusted.gpg.d/
+        rm microsoft-prod.list microsoft.gpg
+        sudo apt-get update
+        sudo apt-get install -y moby-engine
```

Comment:
Sidetrack: 
Do know how if there is a way we can have these command be pulled from the Cx facing doc? 
Too many times that I personally think... I wish I can just pull this snippet of code from the doc and just run.